src/mainboard/clevo/mtl-h/devicetree.cb

```diff
@@ -208,38 +210,17 @@ chip soc/intel/meteorlake
 				device generic 0 on end
 			end
 		end
-		device ref i2c0 on # Touchpad
-			register "serial_io_i2c_mode[PchSerialIoIndexI2C0]" = "PchSerialIoPci"
-			chip drivers/i2c/hid
-				register "generic.hid" = ""ELAN0412""
-				register "generic.desc" = ""ELAN Touchpad""
-				register "generic.irq_gpio" = "ACPI_GPIO_IRQ_LEVEL_LOW(GPP_B00)"
-				register "generic.detect" = "1"
-				register "hid_desc_reg_offset" = "0x01"
-				device i2c 15 on end
-			end
-			chip drivers/i2c/hid
-				register "generic.hid" = ""FTCS1000""
-				register "generic.desc" = ""FocalTech Touchpad""
-				register "generic.irq_gpio" = "ACPI_GPIO_IRQ_LEVEL_LOW(GPP_B00)"
-				register "generic.detect" = "1"
-				register "hid_desc_reg_offset" = "0x01"
-				device i2c 38 on end
-			end
-		end
 		device ref i2c1 on # USB-PD EEPROM
 			register "serial_io_i2c_mode[PchSerialIoIndexI2C1]" = "PchSerialIoPci"
 		end
 		device ref i2c3 on # Pantone ROM
 			register "serial_io_i2c_mode[PchSerialIoIndexI2C3]" = "PchSerialIoPci"
 		end
-		device ref pcie_rp5 on # GLAN
-			register "pcie_rp[PCH_RP(5)]" = "{
-				.clk_src = 2,
-				.clk_req = 2,
-				.flags = PCIE_RP_LTR | PCIE_RP_CLK_SRC_UNUSED,
-			}"
-			register "pcie_clk_config_flag[2]" = "PCIE_CLK_LAN"
+		device ref i2c4 on # ANX7443 USB-C Retimer
```

Comment:
Only on dGPU models, please move to overridetree (if it's not already there)

Reply:
Right, my bad

Reply:
Removed: https://github.com/Dasharo/coreboot/pull/557/commits/ce4e8e305b2aaa4220583324be7d323b800d48cf#diff-3d93912120f7f4c466fd40100f6e49ad0e915222a34652442b4ec95349947ecfR218